QAT/centos-7/media/nginx/Dockerfile

```diff
@@ -98,29 +98,16 @@ RUN cd /opt/build && \
     make install && \
     (mkdir -p /usr/local/lib64/pkgconfig && mv /usr/local/ssl/lib/pkgconfig/*.pc /usr/local/lib64/pkgconfig/)
 
-RUN yum install -y wget cmake3 make python devtoolset-9
-
-
-ARG QAT_CRYPTOMB_REPO=https://github.com/intel/ipp-crypto/archive/ippcp_2020u3.tar.gz
-RUN cd /opt/build && \
-    wget -O - ${QAT_CRYPTOMB_REPO} | tar xz && \
-    mkdir -p ipp-crypto-ippcp_2020u3/sources/ippcp/crypto_mb/build && \
-    cd ipp-crypto-ippcp_2020u3/sources/ippcp/crypto_mb/build && \
-    (. /opt/rh/devtoolset-9/enable && CFLAGS="-Wl,-rpath=/usr/local/ssl/lib" cmake3 -DOPENSSL_INCLUDE_DIR=/usr/local/ssl/include -DOPENSSL_LIBRARIES=/usr/local/ssl/lib -DOPENSSL_ROOT_DIR=/usr/local/ssl .. && \
-    make -j8 )  && \
-    make install && \
-    make install DESTDIR=/opt/dist
-
 RUN yum install -y wget make gcc gawk autoconf automake libtool pkg-config
 
 # load qat-engine
-ARG QAT_ENGINE_REPO=https://github.com/intel/QAT_Engine/archive/v0.6.4.tar.gz
+ARG QAT_ENGINE_REPO=https://github.com/intel/QAT_Engine/archive/v0.6.6.tar.gz
 RUN cd /opt/build && \
     wget -O - ${QAT_ENGINE_REPO} | tar xz && \
     cd QAT_Engine* && \
     ./autogen.sh && \
     export PERL5LIB="$(ls -1 -d /opt/build/openssl-*)" && \
-    ./configure --with-qat_dir=/opt/intel/QAT --with-openssl_dir="$PERL5LIB" --with-openssl_install_dir=/usr/local/ssl --prefix=/opt/intel/QATengine --disable-qat_ecx --with-cc-opt="-DQAT_DISABLE_NONZERO_MEMFREE" --enable-multibuff_offload --enable-multibuff_ecx && \
+    ./configure --with-qat_dir=/opt/intel/QAT --with-openssl_dir="$PERL5LIB" --with-openssl_install_dir=/usr/local/ssl --prefix=/opt/intel/QATengine --disable-qat_ecx --with-cc-opt="-DQAT_DISABLE_NONZERO_MEMFREE -IBUILD_HOME/ipp-crypto-QAT_CRYPTOMB_VER/sources/ippcp/crypto_mb/include" --enable-qat_hw --with-qat_hw_dir=/opt/intel/QAT && \
```

Comment:
-IBUILD_HOME/ipp-crypto-QAT_CRYPTOMB_VER/sources/ippcp/crypto_mb/include" should not be there as ipp-crypto removed